### ISSUE TITLE
Update parameter naming for concat primitives to match _fit_multi_produce

### ISSUE THREAD

**Issue by orionw** [usc-isi-i2/dsbox-cleaning#65]
When we use try to use HorizontalConcat or VerticalConcat it responds with `TypeError: set_training_data() missing 2 required keyword-only arguments: 'inputs1' and 'inputs2'`

We traced this down to line 126 in VerticalConcat.py (similar code in other primitives) where it turns `inputs1` into `inputs` and `inputs2` into`outputs`.  

This is the line: 
`return self._fit_multi_produce(produce_methods=produce_methods, timeout=timeout, iterations=iterations, inputs=inputs1, outputs=inputs2)`

It fails because `set_training_data` requires the inputs names `input1` and `input2` but it's getting `inputs` and `outputs`.

Updating the parameter names should solve the problem.
We can attach a sample pipeline if that would help.

Thanks!

**Comment by mitar** [comment]
No, you just have to implement multi_produce as well, if you add any additional arguments. See [example here](https://gitlab.com/datadrivendiscovery/common-primitives/blob/master/common_primitives/horizontal_concat.py#L75).

**Comment by mitar** [comment]
So no need to rename arguments, just more methods have to be subclassed.
